extra/docs_landing.html

```diff
@@ -48,335 +48,195 @@
+              Search
+            </h3>
+            <p>Build custom applications with your data using Elasticsearch.</p>
+          </div>
+        </a>
+      </div>
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/observability/current/index.html">
+          <div class="card h-100">
+            <h3 class="mt-3">
+              <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltaa08b370a00bbecc/634d9da14e565f1cdce27f7c/observability-logo-color-32px.png');">
+              </span>
+              Observability
+            </h3>
+            <p>Monitor hosts by collecting logs and metrics with Elastic Observability.</p>
+          </div>
+        </a>
+      </div>
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/security/current/es-overview.html">
+          <div class="card h-100">
+            <h3 class="mt-3">
+              <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt5e0e0ad9a13e6b8c/634d9da18473831f96bbdf1e/security-logo-color-32px.png');">
+              </span>
+              Security
+            </h3>
+            <p>Detect, investigate, and respond to threats with Elastic Security for SIEM.</p>
+          </div>
+        </a>
+      </div>
     </div>
-    <div class="col-md-4 col-12 mb-2">
-      <a class="no-text-decoration" href="https://www.elastic.co/guide/en/observability/current/observability-get-started.html">
+  </div>
+
+  <div role="tabpanel" id="product-tab-contents" aria-labelledby="product-tab" tabindex="0" hidden="">
+    <div class="row">
+      <div class="col-md-4 col-12 mb-2">
+      <a class="no-text-decoration" href="en/elasticsearch/reference/current/index.html">
         <div class="card h-100">
-          <h4 class="mt-3">
-            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltaa08b370a00bbecc/634d9da14e565f1cdce27f7c/observability-logo-color-32px.png');"></span>
-            Observability
-          </h4>
-          <p>Monitor hosts by collecting logs and metrics with Elastic Observability.</p>
+          <h3 class="mt-3">
+            <span class="inline-block float-left icon mr-2" style="background-image: url(
+              'https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt36f2da8d650732a0/5d0823c3d8ff351753cbc99f/logo-elasticsearch-32-color.svg');"></span>
+            Elasticsearch
+          </h3>
+          <p>The core distributed, JSON-based search and analytics engine at the heart of the Elastic platform.</p>
```

Comment:
```suggestion
          <p>The core distributed search and analytics engine at the heart of the Elastic platform.</p>
```
We could say "RESTful", like on https://www.elastic.co/elasticsearch/, but I think we don't need it.